Build failure: azahar

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

- Build azahar against `nixos-unstable` (e.g., `nix build github:NixOS/nixpkgs/nixos-unstable#azahar`)

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/306002907

### Relevant log output

```console
[ 22%] Building CXX object externals/spirv-tools/source/CMakeFiles/SPIRV-Tools-static.dir/opcode.cpp.o
In file included from /build/source/externals/spirv-tools/source/opcode.cpp:38:
/build/source/build/externals/spirv-tools/core.insts-unified1.inc:509:194: error: 'SPV_OPERAND_TYPE_OPTIONAL_TENSOR_OPERANDS' was not declared in this scope; did you mean 'SPV_OPERAND_TYPE_OPTIONAL_MEMORY_ACCESS'?
  509 |   {"TensorReadARM", spv::Op::OpTensorReadARM, 0, nullptr, 1, pygen_variable_caps_TensorsARM, 5, {SPV_OPERAND_TYPE_TYPE_ID, SPV_OPERAND_TYPE_RESULT_ID, SPV_OPERAND_TYPE_ID, SPV_OPERAND_TYPE_ID, SPV_OPERAND_TYPE_OPTIONAL_TENSOR_OPERANDS}, 1, 1, 0, nullptr, 0xffffffffu, 0xffffffffu},
      |                                                                                                                                                                                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                                                  SPV_OPERAND_TYPE_OPTIONAL_MEMORY_ACCESS
/build/source/build/externals/spirv-tools/core.insts-unified1.inc:510:163: error: 'SPV_OPERAND_TYPE_OPTIONAL_TENSOR_OPERANDS' was not declared in this scope; did you mean 'SPV_OPERAND_TYPE_OPTIONAL_MEMORY_ACCESS'?
  510 |   {"TensorWriteARM", spv::Op::OpTensorWriteARM, 0, nullptr, 1, pygen_variable_caps_TensorsARM, 4, {SPV_OPERAND_TYPE_ID, SPV_OPERAND_TYPE_ID, SPV_OPERAND_TYPE_ID, SPV_OPERAND_TYPE_OPTIONAL_TENSOR_OPERANDS}, 0, 0, 0, nullptr, 0xffffffffu, 0xffffffffu},
      |                                                                                                                                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                                                                                                                                   SPV_OPERAND_TYPE_OPTIONAL_MEMORY_ACCESS
make[2]: *** [externals/spirv-tools/source/CMakeFiles/SPIRV-Tools-static.dir/build.make:247: externals/spirv-tools/source/CMakeFiles/SPIRV-Tools-static.dir/opcode.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2131: externals/spirv-tools/source/CMakeFiles/SPIRV-Tools-static.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 22%] Building CXX object src/common/CMakeFiles/citra_common.dir/linux/gamemode.cpp.o
[ 22%] Linking CXX static library libcitra_common.a
[ 22%] Built target citra_common
make: *** [Makefile:146: all] Error 2
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.16.1-zen1, NixOS, 25.11 (Xantusia), 25.11.20250819.2007595`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.4`
 - nixpkgs: `/nix/store/pxnx7dbr0pz3qpgw2r315wdbb00afdw7-source`

### Notify maintainers

@arthsmn 
@marcin-serwin 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/az/azahar/package.nix
+++ b/pkgs/by-name/az/azahar/package.nix
@@ -27,7 +27,6 @@
   python3,
   robin-map,
   SDL2,
-  spirv-headers,
   soundtouch,
   stdenv,
   vulkan-headers,
@@ -102,7 +101,6 @@ stdenv.mkDerivation (finalAttrs: {
     qt6.qttools
     soundtouch
     SDL2
-    spirv-headers
     vulkan-headers
     xbyak
 
@@ -145,6 +143,7 @@ stdenv.mkDerivation (finalAttrs: {
     (cmakeBool "DISABLE_SYSTEM_LODEPNG" true)
     (cmakeBool "DISABLE_SYSTEM_VMA" true)
     (cmakeBool "DISABLE_SYSTEM_ZSTD" true)
+    (cmakeBool "DISABLE_SYSTEM_SPIRV_HEADERS" true)
     (cmakeBool "ENABLE_QT_TRANSLATION" enableQtTranslations)
     (cmakeBool "ENABLE_CUBEB" enableCubeb)
     (cmakeBool "USE_DISCORD_PRESENCE" useDiscordRichPresence)

